integrated_channels/integrated_channel/models.py

```diff
@@ -1,13 +1,15 @@
 """
 Database models for Enterprise Integrated Channel.
 """
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
 
+from enterprise.models import EnterpriseCustomer
```

Comment:
@mattdrayer is there an example you can point me to for an alternative to doing this?